frontends/web/src/locales/bg/app.json

```diff
@@ -808,8 +808,8 @@
   },
   "loading": "зареждане...",
   "notification": {
-    "newTxs": "Нова транзакция в {{accountName}}",
-    "newTxs_plural": "{{count}} нови транзакции в {{accountName}}"
+    "newTxs_one": "Нова транзакция в {{accountName}}",
```

Comment:
is `_one` required?

Reply:
Yes in the new v4 plural format https://www.i18next.com/translation-function/plurals